README.md

```diff
@@ -7,3 +7,14 @@ Building blocks for eye tracking in Augmented Reality `AR` and Virtual Reality `
 Chat with the hmd-eyes community here [![Gitter](https://badges.gitter.im/pupil-labs/hmd-eyes.svg)](https://gitter.im/pupil-labs/hmd-eyes?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
 Join the [google group](https://groups.google.com/forum/#!forum/hmd-eyes) to discuss ideas and stay updated. 
+
+## Pre-release v0.1.0
+Unity Integration Package
+
+## Getting Started
+
+- Download Unity3D
+- In the downloaded package, go to Scenes folder and open main.unity
+- Before playing make sure to configure the IP and PORT numbers
+- Select **PupilListener** in the Hierarchy tab
+- Configure the wifi **IP** and **PORT** numbers from the computer running Pupil Service on the Inspector tab 	
```

Comment:
Edit -- Open the Inspector tab. Set the **IP** and **PORT** to the address and port of the computer running `Pupil Service` or `Pupil Capture`.

Add point -- Switch to `Game` mode by pressing the play button in Unity3d interface.